src/server/game/Maps/Map.h

```diff
@@ -484,6 +484,12 @@ class Map : public GridRefMgr<NGridType>
 
     typedef MapRefMgr PlayerList;
     [[nodiscard]] PlayerList const& GetPlayers() const { return m_mapRefMgr; }
+    uint32 GetPlayersInAreaCount(uint32 areaId) const
+        {
```

Comment:
Extra spaces for the curly brackets and everything inside of it. The curly brackets should be at the start of the definition.

```
uint32 GetPlayers.......
{
}
```